src/applications/personalization/dashboard-2/tests/e2e/upcoming-downtime.cypress.spec.js

```diff
@@ -0,0 +1,84 @@
+import disableFTUXModals from '~/platform/user/tests/disableFTUXModals';
+import { mockUser } from '@@profile/tests/fixtures/users/user.js';
+import serviceHistory from '@@profile/tests/fixtures/service-history-success.json';
+import fullName from '@@profile/tests/fixtures/full-name-success.json';
+import disabilityRating from '@@profile/tests/fixtures/disability-rating-success.json';
+
+import manifest from 'applications/personalization/dashboard/manifest.json';
+
+import { mockFeatureToggles } from './helpers';
+
+/**
+ *
+ * @param {boolean} mobile - test on a mobile viewport or not
+ *
+ * This helper:
+ * - loads the my VA Dashboard,
+ *   checks that focus is managed correctly, and performs an aXe scan
+ */
+describe('The My VA Dashboard', () => {
+  beforeEach(() => {
+    disableFTUXModals();
+    cy.login(mockUser);
+    cy.intercept('/v0/profile/service_history', serviceHistory);
+    cy.intercept('/v0/profile/full_name', fullName);
+    cy.intercept(
+      '/v0/disability_compensation_form/rating_info',
+      disabilityRating,
+    );
+    mockFeatureToggles();
+  });
+  it('should show a dismissible modal if a dependent service has downtime approaching in the next hour', () => {
+    // start time is one minute from now
+    const startTime = new Date(Date.now() + 60 * 1000);
+    // end time is one hour from now
+    const endTime = new Date(Date.now() + 60 * 60 * 1000);
+    cy.route('GET', '/v0/maintenance_windows', {
```

Comment:
I need to do `cy.route` instead of `cy.intercept` because of this `cy.route` that's run before every test: https://github.com/department-of-veterans-affairs/vets-website/blob/0e1e0f3caa19fb2b2b5c4be3ed93523857c10060/src/platform/testing/e2e/cypress/support/index.js#L39-L41